orchestrate-cockroachdb-with-docker-swarm.md

```diff
@@ -219,7 +257,7 @@ A secure CockroachDB cluster uses TLS certificates for encrypted inter-node and
    - `--mount`: This flag mounts a local volume called `cockroachdb-1`. This means that data and logs for the node running in this container will be stored in `/cockroach/cockroach-data` on the instance and will be reused on restart as long as restart happens on the same instance, which is not guaranteed.
      {{site.data.alerts.callout_info}}If you plan on replacing or adding instances, it's recommended to use remote storage instead of local disk. To do so, <a href="https://docs.docker.com/engine/reference/commandline/volume_create/">create a remote volume</a> for each CockroachDB instance using the volume driver of your choice, and then specify that volume driver instead of the <code>volume-driver=local</code> part of the command above, e.g., <code>volume-driver=gce</code> if using the <a href="https://github.com/mcuadros/gce-docker">GCE volume driver</a>.
    - `--stop-grace-period`: This flag sets a grace period to give CockroachDB enough time to shut down gracefully, when possible.
-   - `--secret`: These flags identify the secrets to use in securing the node. They must reference the secret names defined in step 5.5. For the node cert and key secrets, the `source` field identifies the relevant secret, and the `target` field defines a more generic name to be used in `cockroach start` flags. For the node key secret, the `mode` field also sets the file permissions to `0600`; if this isn't set, Docker will assign a default file permission of `0400`, which will not work with CockroachDB's built-in SQL client.
+   - `--secret`: These flags identify the secrets to use in securing the node. They must reference the secret names defined in step 5. For the node certificate and key secrets, the `source` field identifies the relevant secret, and the `target` field defines the name to be used in `cockroach start` flags. For the node key secret, the `mode` field also sets the file permissions to `0600`; if this isn't set, Docker will assign a default file permission of `0400`, which will not work with CockroachDB's built-in SQL client.
```

Comment:
This definitely used to be necessary. I'll retest without changing the file permissions, but probably after merging this.